java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/InitializationData.java

```diff
@@ -83,20 +84,18 @@ public InitializationData clone() {
     public BatchRequestInterceptor batchRequestInterceptor;
 
     /**
-     * The SSL engine factory used to configure the client-side ssl transport. If non-null all the
+     * The SSL engine factory used to configure the client-side ssl transport. If non-null, all the
      * IceSSL configuration properties are ignored, and any SSL configuration must be done through
      * the SSLEngineFactory.
      */
     public SSLEngineFactory clientSSLEngineFactory;
 
     /**
-     * The plug-in factories. The corresponding plug-ins are created during communicator initialization,
+     * A list of plug-in factories. The corresponding plug-ins are created during communicator initialization,
      * in order, before all other plug-ins.
      */
     public List<PluginFactory> pluginFactories = Collections.emptyList();
 
-    /**
-     * The Slice loader, used to unmarshal Slice classes and exceptions.
-     */
+    /** The Slice loader, used to unmarshal Slice classes and exceptions. */
```

Comment:
We should clarify the relation with the class loader above, for example the class loader description mentions is used to load class instances and exceptions. My understanding is the Slice loader takes precedence and the class loader will not be used when you provide the Slice loader. I think it would be helpful to clarify this for users.